### PR TITLE
VIDLA-3567 Fixed typo in debug code

### PR DESCRIPTION
**Pull request by graphicdeveloper**
This fixes a minor typo in console debug code and a test script

### CODE REVIEW

**Review by ynisman — APPROVED** [review]

**Comment by graphicdeveloper** [comment]
I just updated this PR to merge back into the 1.2.9+ branch, not master - sorry for the goof!

File: test/ads/vast/VASTClient.spec.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ describe("VASTClient", function () {
         sinon.assert.calledWith(vast._requestVASTXml, 'http://fake.url');
         flushVASTXmlRequest(null, vastXML('<Ad><Wrapper><VASTAdTagURI><![CDATA[http://VASTAdTagURI.com]]></VASTAdTagURI></Wrapper></Ad>'));
 
-        assertError(callback, 'on VASTClient.getVASTAd.getAd, players wrapper limit reached (the limit is 1)', 302);
+        assertError(callback, 'on VASTClient.getVASTAd.getAd, player's wrapper limit reached (the limit is 1)', 302);
```
Comment (vpitts) [inline test/ads/vast/VASTClient.spec.js]: you need to escape the single quote here

**Review by vpitts — CHANGES_REQUESTED** [review]

**Comment by vpitts** [comment]
No worries

_________________________________
Vikki Pitts
Engineering Manager
AppNexus Inc.
17835 Ventura Blvd. #100
Encino, CA 91316
Office: 747.254.7190
Fax: 747.254.7192
Email: vpitts@appnexus.com

**Comment by graphicdeveloper** [comment]
I believe the bug you pointed to was corrected in my 2nd commit to this PR, @vpitts. Please verify, or let me know if I've overlooked something else. Thanks!

File: test/ads/vast/VASTClient.spec.js (hunk 1 above)
Comment (graphicdeveloper) [reply test/ads/vast/VASTClient.spec.js]: I believe the bug you pointed to was corrected in my 2nd commit to this PR, @vpitts. Please verify, or let me know if I've overlooked something else. Thanks!

Comment (vpitts) [reply test/ads/vast/VASTClient.spec.js]: I see the fix. 

However, I notice that is some places the string is wrapped in double quotes (which removes the requirement for the escape character) and in other places the string is wrapped in single quotes.  Wouldn't it better to do it the same in each place.  @ynisman  mentioned that LINT prefers the use of single quotes.  I just seems like less confusion if we do it the same in all places.

**Review by vpitts — APPROVED** [review]

**Pull request merged**